fusedrug/data/tokenizer/ops/modular_tokenizer_ops.py

```diff
@@ -241,22 +243,30 @@ def __call__(
                 )
 
         if isinstance(data, str):
-            encoded, overflow_info = self._tokenizer.encode(
+            _ans = self._tokenizer.encode(
```

Comment:
why "_" prefix?

Reply:
to hint that it's "private" or "local" - and should not be used as the answer to outside,
just a convention - it's possible that it's a convention I only have with myself :D